### ISSUE TITLE
Optimize VS Code when screen reader detected

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#95951]
Simial to how we will hide tabs when we detect a screen reader we could do other optimizations. For example:
* Do not open Settings UI, but settings.json

**Comment by MarcoZehe** [comment]
I actually don't think opening settings.json instead of the settings pane is a good idea. While many items are self-explanatory, others might have more or better verbiage in the actual settings pane. Also, the Settings pane allows for better searching. That would be an "optimization" I would not be happy with. :-)

**Comment by isidorn** [comment]
@MarcoZehe that is perfectly fair. Then I think we should probably improve some of the accessibility issues that the Settings pane faces today 

fyi @roblourens

**Comment by alex19EP** [comment]
> Do not open Settings UI, but settings.json

absolutely against it. I use json presentation myself, but this should remain a conscious choice.

The settings interface has certain accessibility issues, and they really should be fixed.

**Comment by webczat** [comment]
+1. settings.json by default is not a good thing. Also I am generally against forcing settings to be set in a certain way when accessibility is enabled, like some form of optout should be allowed in my opinion, even if defaults are explicitly customized because of screenreader being detected.

**Comment by webczat** [comment]
The example of this is folding. I believe I've heard/read somewhere that some time ago folding was disabled when screenreader was detected. I am not sure whether this is true because even enabled folding doesn't autofold without extensions it seems. I know it works now, but what I meant is, that you couldn't override that choice. Of course you can report each such case, but then you need to wait till the case is resolved.

**Comment by isidorn** [comment]
Any change we due to optmise it will be possible to go back to the previous behavior.
But getting your feedback: we will not change the behavior of settings.
So for now will only disable tabs (which can of course be brought back via settings). Will let you know if I have more ideas

**Comment by webczat** [comment]
nice.

**Comment by alex19EP** [comment]
ok i don't know whether it’s necessary to create yet another issue but it would be great if the indicator about accessibility in the status bar could be turned off.

**Comment by isidorn** [comment]
@alex19EP makes perfect sense, José already filed this issue so let's tackle that there https://github.com/microsoft/vscode/issues/96210

**Comment by akash07k** [comment]
Thanks @isidorn For not forcing the settings.json upon screen reader detection. It would be amazing if the issues with accessibility pane will be fixed.

**Comment by isidorn** [comment]
Thanks a lot for feedback. Due to the above we will not do any changes, except maybe the one for tabs which is captured via this item https://github.com/microsoft/vscode/issues/95682
Thus closing this.
